data/avro-schema/src/main/resources/avro/net/corda/data/flow/state/checkpoint/FlowState.avsc

```diff
@@ -0,0 +1,61 @@
+{
+  "type": "record",
+  "name": "FlowState",
+  "namespace": "net.corda.data.flow.state.checkpoint",
+  "doc": "State used by the flow engine to track the execution progress of the flow",
+  "fields": [
+    {
+      "name": "fiber",
+      "type": "bytes",
+      "doc": "Flow execution state, serialized at the point the flow was suspended."
+    },
+    {
```

Comment:
maybe a bit pedantic, but do we want the immutable flow start context mixed with the mutable flow state? should the start context live on the root of the checkpoint?